pkg/apis/certmanager/v1/types_certificate.go

```diff
@@ -433,6 +433,14 @@ type CertificateStatus struct {
 	// not set or False.
 	// +optional
 	NextPrivateKeySecretName *string `json:"nextPrivateKeySecretName,omitempty"`
+
+	// The number of times certificate issuance has been attempted and
+	// failed for this revision (see the description of status.revision
+	// field). If an issuance has failed, the delay till the next issuance
+	// will be calculated using formula time.Hour * 2 ^ (issuanceAttempts - 1).
+	// This field will only be set if the issuance is currently failing.
+	// +optional
```

Comment:
```suggestion
	// +kubebuilder:validation:Minimum=1
	// +optional
```

Reply:
We cannot use the minimum,maximum fields at the moment due to the helm chart conversion bug https://github.com/cert-manager/cert-manager/pull/3647
I seem to remember that we will be able to do it once the oldest cert-manager supported kube version will be 1.20, but cannot find a confirmation to this right now